### PR TITLE
Add asciinema to PuppetLintBear and SpellCheckBear

### PR DESCRIPTION
**Pull request by gs0510**
added an asciinema recording to PuppetLintBear.py as mentioned in the issue #784  and SpellCheckBear.py as mentioned in the issue #785  .

### CODE REVIEW

File: bears/configfiles/PuppetLintBear.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ class PuppetLintBear:
     AUTHORS = {'The coala developers'}
```
Comment (Adrianzatreanu) [inline bears/configfiles/PuppetLintBear.py]: 1stly, in commit body its "Adds"

Comment (Adrianzatreanu) [inline bears/configfiles/PuppetLintBear.py]: its ``Closes``, not Closes issue.

File: bears/natural_language/SpellCheckBear.py [hunk 2]
```diff
@@ -17,6 +17,7 @@ class SpellCheckBear:
     AUTHORS = {'The coala developers'}
```
Comment (Adrianzatreanu) [inline bears/natural_language/SpellCheckBear.py]: same here, make it Adds.

**Comment by gs0510** [comment]
@Adrianzatreanu Made the corrections to the commits.

**Comment by sils1297** [comment]
Hi,

We just dployed a fix to what's causing your builds to fail - you can get thm on your branch with a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*) The rebase should fix the build!

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala/coala-bears.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by Udayan12167** [comment]
ack 30a28c2

**Comment by Udayan12167** [comment]
ack 7d8a8fb

**Comment by Udayan12167** [comment]
@rultor merge

**Comment by sils1297** [comment]
I think somebody else was merged in the meantime and another rebase is
needed, quick and painless

**Comment by SanketDG** [comment]
ack eaf9c54 7f4af09

**Comment by SanketDG** [comment]
ack 7f4af09

**Comment by SanketDG** [comment]
@rultor merge

**Pull request merged**